spec/ewb/IEC61970/Base/Core/Curve.yaml

```diff
@@ -0,0 +1,10 @@
+name: Curve
+description: A multi-purpose curve or functional relationship between an independent variable (X-axis) and dependent (Y-axis) variables.
+attributes:
+- name: curveData
+  type: CurveData
+  description: The point data values that define this curve.
+ancestors:
+- IdentifiedObject
+descendants:
+- ReactiveCapabilityCurve
```

Comment:
add newlines to end of all these files plz

Reply:
done